packages/utils/src/common/array-unique.ts

```diff
@@ -0,0 +1,5 @@
+type ArrayUniqueElement = string | number
+
+export function arrayUnique<T = ArrayUniqueElement>(array: T[]): T[] {
```

Comment:
todo: we have an util call `deduplicate` which does exactly that. I suggest to remove that one